sparse/tests/test_coo.py

```diff
@@ -1527,3 +1527,35 @@ def test_invalid_iterable_error():
     with pytest.raises(ValueError):
         x = [((2.3, 4.5), 3.2)]
         COO.from_iter(x)
+
+
+def test_roll_coo():
```

Comment:
Optional: Maybe follow the pattern of using `sparse.random` to generate something of a given shape and then roll it using both NumPy and sparse implementations.

Reply:
Mandatory: Parameterize your tests. There are a lot of examples in the tests already, and [docs here](https://docs.pytest.org/en/latest/parametrize.html).